Improve content metadata field docs

justinmayer opened the pull request:
Fixes #2347

oulenz commented on docs/content.rst:
```diff
@@ -71,22 +71,31 @@ Metadata syntax for Markdown posts should follow this pattern::
+``:status:``      ``Status:``      Content status:
+                                   ``draft``, ``hidden``, or ``published``
+``:template:``    ``Template:``    Name of template to use to generate content
+``:save_us:``     ``Save_us:``     Save content to this relative file path
```
should be `save_as`

justinmayer replied:
Fixed in https://github.com/getpelican/pelican/pull/2620/commits/f0617a53dc253432077fe3acbcd8d8bd87f636ab

oulenz commented on docs/content.rst:
```diff
@@ -71,22 +71,31 @@ Metadata syntax for Markdown posts should follow this pattern::
+``:summary:``     ``Summary:``     Brief description of content for index pages
+``:lang:``        ``Lang:``        Content language ID (``en``, ``fr``, etc.)
+``:translation:`` ``Translation:`` Is article is a translation of another
+                                   (``true`` or ``false``)
```
`Is article is` -> `Is content`

justinmayer replied:
Fixed in https://github.com/getpelican/pelican/pull/2620/commits/f0617a53dc253432077fe3acbcd8d8bd87f636ab

oulenz commented on docs/content.rst:
```diff
@@ -71,22 +71,31 @@ Metadata syntax for Markdown posts should follow this pattern::
 
 You can also have your own metadata keys (so long as they don't conflict with
-reserved metadata keywords) for use in your python templates. The following is
+reserved metadata keywords) for use in your templates. Following is
```
I think `Following is` -> `This is` would parse easier

justinmayer replied:
Didn't quite agree that was clearer, so I found another phrasing that might be easier to parse in https://github.com/getpelican/pelican/pull/2620/commits/f0617a53dc253432077fe3acbcd8d8bd87f636ab.

justinmayer commented on docs/content.rst:
```diff
@@ -71,22 +71,31 @@ Metadata syntax for Markdown posts should follow this pattern::
+================= ================ ============================================
+``:title:``       ``Title:``       Title of the article or page
+``:date:``        ``Date:``        Publication date (``YYYY-MM-DD HH:SS``)
+``:modified:``    ``Modified:``    Modification date (``YYYY-MM-DD HH:SS``)
```
Understood, but there isn't a lot of room in that table to explain that, and the contents of the docs in that link are nearly inscrutable to a normal human being. I prefixed the example syntax with `e.g.,` to at least hint that there are other formats available. If someone wants to make a follow-up PR to expand upon this and elaborate on the available date/time formats provided by `dateutil`, we'll get it merged. I can see how that would be helpful.

justinmayer commented on docs/content.rst:
```diff
@@ -71,22 +71,31 @@ Metadata syntax for Markdown posts should follow this pattern::
+``:date:``        ``Date:``        Publication date (``YYYY-MM-DD HH:SS``)
+``:modified:``    ``Modified:``    Modification date (``YYYY-MM-DD HH:SS``)
+``:tags:``        ``Tags:``        Content tags, separated by commas
+``:keywords:``    ``Keywords:``    Content keywords, separated by commas
```
Quite right. I added a note explaining it applies to HTML source content only.

justinmayer commented on docs/content.rst:
```diff
@@ -71,22 +71,31 @@ Metadata syntax for Markdown posts should follow this pattern::
+                                   (``true`` or ``false``)
+``:status:``      ``Status:``      Content status:
+                                   ``draft``, ``hidden``, or ``published``
+``:template:``    ``Template:``    Name of template to use to generate content
```
Fixed in https://github.com/getpelican/pelican/pull/2620/commits/f0617a53dc253432077fe3acbcd8d8bd87f636ab

justinmayer commented:
Many thanks to @oulenz and @avaris for the feedback on this PR. To the best of my knowledge, I addressed all the comments. Let me know if there's anything further I should change before merging.
